MediaBrowser.Model/Session/HardwareEncodingType.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace MediaBrowser.Model.Session
+{
+    /// <summary>
+    /// Enum HardwareEncodingType.
+    /// </summary>
+    public enum HardwareEncodingType
+    {
+        /// <summary>
+        /// AMD AMF
+        /// </summary>
+        AMF,
```

Comment:
Please also set the value for all enums
```suggestion
        AMF = 1,
```